shims/0.23/src/main/java/org/apache/hadoop/hive/shims/Hadoop23Shims.java

```diff
@@ -1223,11 +1223,10 @@ public static boolean isHdfsEncryptionSupported() {
     private final Configuration conf;
 
     public HdfsEncryptionShim(URI uri, Configuration conf) throws IOException {
-      DistributedFileSystem dfs = (DistributedFileSystem)FileSystem.get(uri, conf);
-
       this.conf = conf;
-      this.keyProvider = dfs.getClient().getKeyProvider();
       this.hdfsAdmin = new HdfsAdmin(uri, conf);
+      // HDFS-11687 added getKeyProvider method in hdfsAdmin from Hadoop-2.9.0
```

Comment:
Please remove this comment.  Keep this information in JIRA.  Thanks.

Reply:
Done. Thanks